crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -823,6 +823,64 @@ impl Bounded3d for Triangle3d {
     }
 }
 
+/// A tetrahedron primitive.
+#[derive(Clone, Copy, Debug, PartialEq)]
+#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+pub struct Tetrahedron {
```

Comment:
This should have a `Reflect` impl [here](https://github.com/bevyengine/bevy/blob/main/crates/bevy_reflect/src/impls/math/primitives3d.rs)